src/Collector.cs

```diff
@@ -35,6 +35,10 @@ class Collector
     private bool _queueFullWarned;
     private static readonly ILog _log = LogManager.GetCurrentClassLogger();
 
+    private static readonly string InstrumentalAddress = "collector.instrumentalapp.com";
+    private static readonly int InstrumentalPort = 8000;
+    private static readonly byte[] InstrumentalOk = {111, 107, 10};
```

Comment:
Who knew c# strings-to-bytes was so weird?  The unicode consortium, that's who.

This is an easy one, though!